storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -161,6 +167,3 @@ include::modules/lvms-download-log-files-and-diagnostics.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data[About the must-gather tool]
-
-//Reference
-include::modules/lvms-reference-file.adoc[leveloffset=+1]
```

Comment:
I am curious why you made a bunch of changes to the reference file if you are removing it from the assembly?

Reply:
I have renamed the reference file and added conceptual info.